.github/ISSUE_TEMPLATE/doc-issue.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Doc issue
+about: Create an issue to help us improve
+
+---
+
+**Before you enter an issue**
+
+- If the issue is a simple typo or similar correction, you can just submit a PR. 
```

Comment:
Often folks open an issue about a typo because they don't know how to create a PR. So, the following link in the template might be useful:
https://docs.microsoft.com/en-us/contribute/#quick-edits-to-existing-documents

Reply:
Great suggestion.